config/theme.yml

```diff
@@ -121,6 +121,51 @@ global_settings:
       width: 170
       height: 115
       scope: [stores]
+    default_80:
+      width: 80
+      height: 80
+      scope: [categories, products]
+    default_100:
+      width: 100
+      height: 100
+      scope: [manufacturers, suppliers]
+    default_120:
+      width: 120
+      height: 120
+      scope: [products]
+    default_160:
+      width: 160
+      height: 160
+      scope: [categories, products]
+    default_200:
+      width: 200
+      height: 200
+      scope: [stores, manufacturers, suppliers, products]
+    default_320:
+      width: 320
+      height: 320
+      scope: [categories, products]
+    default_400:
+      width: 400
+      height: 400
+      scope: [stores, products]
+    default_720:
+      width: 720
+      height: 720
+      scope: [products]
+    default_1000:
+      width: 1000
+      height: 1000
+      scope: [categories]
+    default_1440:
+      width: 1440
+      height: 1440
+      scope: [products]
+    default_2000:
+      width: 2000
+      height: 2000
+      scope: [categories]
```

Comment:
I can't imagine where images so big would be used. Maybe when zooming (old jqzoom) products, but definitely not categories.

Reply:
@SharakPL this is the wide image on the category page, only used for high pixel density screens

The regular one is 1000px, but on 2x screens, it will take the 2000px, this is absolutely needed

Reply:
Then it should be `height: 200`, not 2000 😉